src/ripple/protocol/impl/STLedgerEntry.cpp

```diff
@@ -78,7 +78,7 @@ void STLedgerEntry::setSLEType ()
     type_ = mFormat->getType ();
     if (!setType (mFormat->elements))
     {
-        if (auto j = debugLog())
+        if (auto j = debugLog().error())
```

Comment:
@nbougalis: If we wanted to future-proof this solution, I could easily add a `Journal` to `DebugSink`: https://github.com/ripple/rippled/blob/develop/src/ripple/basics/impl/Log.cpp#L348.  Then I would return a reference to that `Journal` instead of a temporary `Journal`.  Would you like me to make that change?